packages/just-scripts/src/tasks/sassTask.ts

```diff
@@ -39,20 +40,19 @@ export function sassTask(
     }
 
     const autoprefixerFn = autoprefixer({ browsers: ['> 1%', 'last 2 versions', 'ie >= 11'] });
-
+    const postcssRtlFn = postcssRtl({});
```

Comment:
This might be undefined if it's not installed. And I don't think we should suddenly start requiring it to be installed, so you should probably add some logic to handle if it's absent.

Reply:
Same with cssnano

Reply:
Agreed - we should make these two new plugins optional and we should have another issue to make the autoprefixer + sass to be optional in another PR.

Reply:
Fixed.